Dwolla.Client/DwollaClient.cs

```diff
@@ -21,9 +21,10 @@ public interface IDwollaClient
         string ApiBaseAddress { get; }
         string AuthBaseAddress { get; }
 
-        Task<RestResponse<TRes>> PostAuthAsync<TReq, TRes>(Uri uri, TReq content);
-        Task<RestResponse<TRes>> GetAsync<TRes>(Uri uri, Headers headers);
-        Task<RestResponse<TRes>> PostAsync<TReq, TRes>(Uri uri, TReq content, Headers headers);
+        Task<RestResponse<TRes>> PostAuthAsync<TReq, TRes>(Uri uri, TReq content) where TRes : IDwollaResponse;
+        Task<RestResponse<TRes>> GetAsync<TRes>(Uri uri, Headers headers) where TRes : IDwollaResponse;
+        Task<RestResponse<TRes>> PostAsync<TReq, TRes>(Uri uri, TReq content, Headers headers) where TRes : IDwollaResponse;

```

Comment:
Should be a test for this new public method

Reply:
Added.